.gitignore

```diff
@@ -32,7 +32,7 @@ pip-log.txt
 .coverage
 .tox
 coverage.xml
-tests/coverage_html_report
```

Comment:
Coverage reports will now be ignored anywhere, not just inside the `tests` folder.